### PR TITLE
Improve header

### PR DESCRIPTION
**Pull request by nnorbert**
Improve header

### CODE REVIEW

**Review by alfonsograziano — APPROVED** [review]
LGTM

**Comment by nnorbert** [comment]
Updated!
Screenshot 2024-04-24 at 13 26 33

File: apps/profile/app/components/FeedbackBanner.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default () => {
             mail: (chunks) => (
               <a
                 className="govie-link"
-                href="mailto:tiago.ramos@nearform.com?Feedback"
+                href="mailto:tiago.ramos@nearform.com?subject=Feedback"
```
Comment (alfonsograziano) [inline apps/profile/app/components/FeedbackBanner.tsx:16]: @GiuliaTeggi @Thurfjell FYI

**Review by alfonsograziano — APPROVED** [review]
LGTM

**Pull request merged**